Cubical/Categories/Displayed/Base/More.agda

```diff
@@ -61,3 +62,23 @@ module _ {C : Category ℓC ℓC'} {D : Category ℓD ℓD'}
   _^opFᴰ .F-homᴰ = Fᴰ .F-homᴰ
   _^opFᴰ .F-idᴰ = Fᴰ .F-idᴰ
   _^opFᴰ .F-seqᴰ fᴰ gᴰ = Fᴰ .F-seqᴰ gᴰ fᴰ
+
+module _
+  {C : Category ℓC ℓC'} {D : Category ℓD ℓD'}
+  (Dᴰ : Categoryᴰ D ℓDᴰ ℓDᴰ') (F : Functor C D)
+  where
+
+  private
+    module C = Category C
+    module D = Category D
+
+  open Categoryᴰ Dᴰ
+  open Functor F
+  open Functorᴰ
+
+  -- TODO: better name?
+  reindexΠ : Functorᴰ F (reindex Dᴰ F) Dᴰ
```

Comment:
maybe "reindex" with a superscript "d" for displays? or maybe "reindexing-functor"?

Reply:
Idk I think `reindexΠ` is more informative than both of those. In terms of the total space of the displayed categories involved, this is a projection.

Reply:
It seems opposite to some of the cubical naming scheme though. Like, `isSetΠ` and other similarly named things are refering to `Π` types

Reply:
then maybe just lower-case pi?

Reply:
I went with "forgetReindex" since it's a kind of forgetful functor